cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js

```diff
@@ -209,37 +215,57 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
         userData.userId = userProperties.userId;
         userData.barcode = userProperties.barcode;
         UserEdit.addServicePointViaApi(testData.userServicePoint.id, userData.userId, testData.userServicePoint.id);
+        cy.getToken(userData.username, userData.password);
         UserLoans.updateTimerForAgedToLost('minute');
+        cy.getAdminToken();
       })
       .then(() => {
-        cy.get('@items').each((item) => {
-          Checkout.checkoutItemViaApi({
-            id: uuid(),
-            itemBarcode: item.barcode,
-            loanDate: moment.utc().format(),
-            servicePointId: testData.userServicePoint.id,
-            userBarcode: userData.barcode,
-          });
-        });
+        cy.login(userData.username, userData.password);
```

Comment:
You can do this in one line
cy.login(user.username, user.password, { path:TopMenu.ordersPath, waiter: Orders.waitLoading });

Reply:
unfortunately it won't work. When you go directly to conditions section, it is not displayed properly